packages/design-system-react-native/src/temp-components/Spinner/Spinner.tsx

```diff
@@ -0,0 +1,72 @@
+import { useTailwind } from '@metamask/design-system-twrnc-preset';
+import React, { useEffect } from 'react';
+import { View } from 'react-native';
+import Animated, {
+  useSharedValue,
+  useAnimatedStyle,
+  withRepeat,
+  withTiming,
+  Easing,
+} from 'react-native-reanimated';
+
+import type { IconProps } from '../../components/Icon';
+import Icon, { IconName } from '../../components/Icon';
+import type { TextProps } from '../../components/Text';
+import Text from '../../components/Text';
+import { DEFAULT_SPINNER_PROPS } from './Spinner.constants';
+import type { SpinnerProps } from './Spinner.types';
+
+const Spinner = ({
```

Comment:
Should we update the docs to remove the `Temp` prefix for temporary components? This one doesn't use the prefix, and I think removing it across the board would make it easier to refactor when moving components from the `temp` folder into the main `components` folder.